app/controllers/application_controller.rb

```diff
@@ -1,4 +1,13 @@
 class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
+	before_action :configure_permitted_parameters, if: :devise_controller?
 
-end
+    protected
+
+        def configure_permitted_parameters
+            devise_parameter_sanitizer.permit(:sign_up) { |u| u.permit(:firstname, :lastname, :email, :password)}
+
+            devise_parameter_sanitizer.permit(:account_update) { |u| u.permit(:firstname, :lastname, :email, :password, :current_password)}
+		end
```

Comment:
can you configure your `sublime` to have two spaces (not tab) indent.

Reply:
is this the setting in user preference in sublimt?
// Settings in here override those in "Default/Preferences.sublime-settings", and
// are overridden in turn by file type specific settings.
{
    "tab_size": 2,
    "translate_tabs_to_spaces": true
}